x-pack/plugins/infra/public/pages/metrics/index.tsx

```diff
@@ -14,15 +14,19 @@ import {
   EuiTitle,
 } from '@elastic/eui';
 import { InjectedIntl, injectI18n } from '@kbn/i18n/react';
+import { GraphQLFormattedError } from 'graphql';
 import React from 'react';
 import styled, { withTheme } from 'styled-components';
 
+import { InfraMetricsErrorCodes } from '../../../server/lib/adapters/metrics/adapter_types';
```

Comment:
While importing from the server that way works for type-level constructs, how about we move that enum into an error catalog in `common`?

That way we remove the risk of running afoul of the Kibana packaging process in case this changes.